.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
+	url = https://github.com/HannesWell/m2e-core-tests.git
+	branch = removeRepo
```

Comment:
> You're right. I just imported the test project into launched IDE and it worked well without the change and the change even broke it. It is reverted.

I can only remember that I have had a hard time with the Archetype stuff as it also tries to resolve things from central, in the end I created a special settings.xml for that tests that uses central. But be aware that this could considerably slow down things as each test seem to use its own (empty) local repo.

Given that these tests never run of the CI, and if you want to finish this, we may simply disable the the failing  test for now?

Reply:
> Given that these tests never run of the CI, and if you want to finish this, we may simply disable the the failing test for now?

I'm torn. As you said 'later' often means never, especially for such an comfortable flaw like an ignored test.
Give me a few more days. If then given up, I will disable them.

Reply:
Sure if you like take your time, I just think we should not block this for too long, and it does not make it worse compared to the previous state, and one could instantly open a PR for fixing the remaining test...